LocalMultiplayerAgent/Config/MultiplayerSettings.cs

```diff
@@ -46,6 +46,8 @@ public class MultiplayerSettings
         public ProcessStartParameters ProcessStartParameters { get; set; }
 
         public SessionConfig SessionConfig { get; set; }
+
+        public int NodePort { get; set; }
```

Comment:
NodePort is a property in the array PortMappingsList, specifically on the PortMapping object. That property is the one we need to validate, right?